src/mixins/filing-template-mixin.ts

```diff
@@ -155,14 +150,13 @@ export default class FilingTemplateMixin extends DateMixin {
       filing.correction.parties = parties
     }
 
-    // add in data specific to corp class BC corrections
-    if (this.isBcCorrectionFiling) {
+    // add in data specific to BEN/BC/CCC/ULC corrections
+    if (this.isBenBcCccUlcCorrectionFiling) {
       filing.correction.nameTranslations = isDraft ? this.getNameTranslations : this.prepareNameTranslations()
       filing.correction.shareStructure = {
         shareClasses: isDraft ? this.getShareClasses : this.prepareShareClasses(),
         resolutionDates: this.getNewResolutionDates
       }
-      filing.correction.provisionsRemoved = this.areProvisionsRemoved
```

Comment:
Pre-existing conditions code now only applies to alterations, so here, in a correction filing object, we don't need to set this.